src/test/resources/InsertIntoTableIntegrationTest/data.tsv

```diff
@@ -0,0 +1,2 @@
+a1	b1	c1	d1	e1
```

Comment:
Any idea why this shows up as new instead of moved?

Reply:
Well it might be because `/InsertIntoTableIntegrationTest/` and `/TsvFileParserTest/` were the same folder that was used by two different test classes, so maybe when I was separating them I deleted some files and recreated them? Might be a mystery we will never solve

Reply:
OK, yes, if you did a delete and recreate then Eclipse or git itself might have lost the connection and not treated it as a "move".